.github/workflows/maestro-android.yml

```
@@ -34,21 +48,28 @@ jobs:
 
       - name: Install Maestro
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:2:30: Double quote to prevent globbing and word splitting

(shellcheck)